<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsDeleteWindow</class>
 <widget class="QDialog" name="ContactsDeleteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="DeleteContactBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>650</width>
     <height>320</height>
    </rect>
   </property>
   <property name="title">
    <string>Delete Contact</string>
   </property>
   <widget class="QGroupBox" name="EventsBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>300</width>
      <height>230</height>
     </rect>
    </property>
    <property name="title">
     <string>Events</string>
    </property>
    <widget class="QTableView" name="EventsTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ContactsForTheEventBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>300</width>
      <height>230</height>
     </rect>
    </property>
    <property name="title">
     <string>Contacts For The Event</string>
    </property>
    <widget class="QTableView" name="ContactsForTheEventTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>610</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteContactButton">
       <property name="text">
        <string>Delete Contact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
